setup.cfg

```diff
@@ -65,16 +68,16 @@ exclude =
 [options.extras_require]
 testing =
   # upstream
-  pytest >= 7
+  pytest >= 7.4
   pytest-checkdocs >= 2.4
   pytest-flake8 >= 1.0.7
-  coverage >= 6.3
+  coverage >= 7
   pytest-enabler >= 1.0.1
   # local
   flake8 >= 5.0
-  virtualenv >= 15.0.1
-  wheel >= 0.37.0
-  pip >= 21.0
+  virtualenv >= 20.26.5
+  wheel >= 0.44.0
+  pip >= 22.0
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.